tests/imap_integration.rs

```diff
@@ -1,18 +1,21 @@
 extern crate imap;
 extern crate lettre;
 extern crate lettre_email;
+#[cfg(feature = "ssl")]
```

Comment:
I think it's probably fine for `native-tls` to always be included in `dev-dependencies`

Reply:
Good pickup, that makes sense.